modules/rubiconBidAdapter.js

```diff
@@ -783,6 +791,63 @@ function _renderCreative(script, impId) {
 </html>`;
 }
 
+function hideGoogleAdsDiv(adUnit) {
+  const el = adUnit.querySelector("div[id^='google_ads']");
+  if (el) {
+    el.style.setProperty('display', 'none');
+  }
+}
+
+function hideSmartAdServerIframe(adUnit) {
+  const el = adUnit.querySelector("script[id^='sas_script']");
+  if (el && el.nextSibling && el.nextSibling.localName === 'iframe') {
+    el.nextSibling.style.setProperty('display', 'none');
+  }
```

Comment:
for better minification i recommend using a `nextSibling` var
```
const nextSibling = el && el.nextSibling;
if (nextSibling && nextSibling.localName === 'iframe') {
    nextSibling.style.setProperty('display', 'none');
}
```

this should result in fewer chars and reduce file size

Reply:
Done.